optionals/headgear/CfgWeapons.hpp

```diff
@@ -0,0 +1,59 @@
+class CfgWeapons {
+    class CUP_H_OpsCore_Green;
+    class CLASS(OpsCore_Medic): CUP_H_OpsCore_Green {
+        author = ECSTRING(main,Authors);
+        displayName = "OpsCore FAST (Medic)";
+        scope = 2;
+        hiddenSelections[] = {
+            "camo1",
+            "camo2",
+            "camo3",
+            "flag"
+        };
+        hiddenSelectionsTextures[] = {
+            QPATHTOF(data\cup_opscore_medic_co.paa),
+            QPATHTOF(data\cup_opscore_medic_1_co.paa),
+            QPATHTOF(data\cup_opscore_medic_2_co.paa),
+            ""
+        };
+    };
+
+    class CUP_H_USA_Cap_AU_DEF;
+    class CLASS(Cap_DEF_NYMets): CUP_H_USA_Cap_AU_DEF {
+        author = "GilleeDoo";
+        scope = 2;
+        displayName = "Cap w/DEF (New York Mets)";
+        hiddenSelectionsTextures[] = {
+            QPATHTOF(data\cap_def_nymets_co.paa)
+        };
+    };
+
+    // Jack
+    class ItemCore;
+    class HeadgearItem;
+    class CUP_H_C_Beret_01: ItemCore {
+        class ItemInfo: HeadgearItem {
+            class HitpointsProtectionInfo {
+                class Head;
+            };
+        };
+    };
+
+    class CLASS(Jack_Beret): CUP_H_C_Beret_01 {
```

Comment:
```suggestion
    class tacgt_special_Jack_Beret: CUP_H_C_Beret_01 {
```
Don't see a reason to change it.

Reply:
Could also do:
```cpp
#define ECLASS(var1,var2) TRIPLES(PREFIX,var1,var2)
#define QECLASS(var1,var2) QUOTE(TRIPLES(PREFIX,var1,var2))
```
and use it as `[Q]ECLASS(Jack_Beret)`.